cgi/import_file_process.pl

```diff
@@ -191,57 +193,24 @@
 
 store("$data_root/import_files/${Owner_id}/import_files.sto", $import_files_ref);
 
-$html .= "<p>" . lang("import_file_status") . lang("sep"). ': <span id="result">' . lang("job_status_inactive") . '</span>';
+$template_data_ref->{job_status_inactive} = lang("job_status_inactive");
```

Comment:
It's better to remove that line, and to call lang("job_status_inactive") directly in the template.